sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ClientConstants.java

```diff
@@ -8,11 +8,11 @@
     public static final String PLATFORM_INFO = getOSInformation();
     public static final String FRAMEWORK_INFO = getFrameworkInfo();
 
-    /**
+    /*
      * Gets the USER AGENT string as defined in:
      * $/core/azure-core/src/main/java/com/azure/core/http/policy/UserAgentPolicy.java
-     * TODO (conniey): Extract logic from UserAgentPolicy into something we can use here.
      */
+    @Deprecated
```

Comment:
This is `implementation` code, do we still need to use `@Deprecated` or could we remove it instead?

Reply:
We shouldn't need Deprecated because it's not part of our API surface area.